### DISCUSSION TITLE
ventoy-1.0.99-windows

### DISCUSSION THREAD

**Discussion by HC1Gunner** [memstechtips/UnattendedWinstall#90] [General]
I did as told, created an ISO folder and Templates folder on the root of my USB D:
When I copy/pasted the directory path in the VentoyPlugson page under Auto Install Plugin menu, I get an error the the 2nd directory for the unattended xml location. "The 2nd file path is invalid or nonexist!".
It does not matter if I add the 2nd Dir as D:\Templates\unattend.xml, it still errors.

It would have been nice if you did an exact example of your own, the suggestions from  Ventoy are both wrong compared to the directions for creating an ISO and Templates folder.
For example:
D:\ISO\Windows
D:\script\Windows_unattend.xml
--------------------------------------------------------------------------------------------------------------------
My directories to each folder.
D:\ISO

D:\Templates

**Comment by memstechtips** [comment]
I will make a tutorial video about this method in the near future.
The instructions were initially written by another contributor and it's possible that there might be some issues.

Thanks for bringing this to my attention.

**Reply by HC1Gunner** [reply]
Any ideas,  are the instructions correct for making (2) folders on the root of the drive named ISO and Templates. Ventoy adds a script to the root "VentoyPlugson". Its Dir example appears to add a script "D:\script\Windows_unattend.xml".

I've tried about every combination for the unattended Dir, and it fails every time.

I really need to do a clean install, currently running Win 11 and its terrible, I get several blue screens per/month. I have an issue where windows just disappears (shuts down) and I left on a blank screen (no video out from PC). When I look at the event viewer, it has no codes, just says windows improperly shutdown. I don't want to, but with such an unstable system I going to swap out my M.2 SSD, and install Win 10. Then swap drives and do a clean unattended install of Win 11.

*** Now you have me a bit confused your article (Unattended Windows Installation) referencing to using the IoT-LTSC xml points users to use Ventoy2Disk to create a bootable USB drive, and VentoyPlugson to add your autounattend file to the install.
Further down this page you list two options (1) Method 1: Create a Bootable Windows Installation, using Rufus to create a Bootable drive from and ISO, and then just copy the autounattend xml file to the root of the drive and you're good to go. If this option works, and does the same thing, then why bother using the (2) Ventoy files to accomplish the same thing?

![ventoy Dir](https://github.com/user-attachments/assets/f92aaa4d-70b3-40a9-80ff-c86730aaab11)

**Reply by memstechtips** [reply]
Hi there,

I will create a video tutorial for Method 3 soon.

All 3 methods are a bit different.
Method 1: For a Single USB drive with either Windows 10 or Windows 11.
Method 2: The `autounattend.xml` file is integrated into the ISO file. You need to recreate the ISO file every time you want to use a new version of the `autounattend.xml` file.
Method 3: The Windows ISO files are never changed or edited. You can just replace the answer file(s) on the drive. It should be much faster and easier compared to the other methods.

**Reply by memstechtips** [reply]
Tutorial video: https://youtu.be/4AGZQJTyCOs
